docker-compose.yml

```diff
@@ -2,13 +2,6 @@
 version: '3.7'
 
 services:
-  db:
```

Comment:
Why not leaving it as default? And overriding only when postpres is used?

Reply:
fixed